lib/msf/core/web_services/sinatra/rpc_json_exceptions.rb

```diff
@@ -0,0 +1,27 @@
+require 'msf/core/rpc'
+
+module Sinatra
+  # A Sinatra extension that allows the customization of errors returned by Sinatra.
+  module RpcJsonExceptions
+
+    def self.registered(app)
+      app.set :show_exceptions => false
+
+      app.error { |err|
+        se = Msf::RPC::JSON::Dispatcher::create_error_response(
```

Comment:
se? server error? I think we can afford a few more letters here to make this a touch clearer

Reply:
Agreed

Reply:
Renamed `se` to `sinatra_error`